LLMObs: update auto-instrumentation version support (#36151)
DataDog/documentation: content/en/llm_observability/instrumentation/auto_instrumentation.md

{{< /tabs >}}
{{% /collapse-content %}}

{{% collapse-content title="Claude Agent SDK" level="h3" expanded=false id="claude-agent-sdk" %}}
{{< tabs >}}
{{% tab "Python" %}}
The Claude Agent SDK integration provides automatic tracing for agent queries made through the [Claude Agent SDK for Python][1].

> ### Traced methods

Reviewer: Every standard integration section on this page includes `**Package name:**` and `**Integration name:**` metadata immediately before `### Traced methods`. The Claude Agent SDK section is missing these fields.

```suggestion
**Package name:** `claude-agent-sdk`
**Integration name:** `claude_agent_sdk`

### Traced methods
```

Is this right?

{{< /tabs >}}
{{% /collapse-content %}}

{{% collapse-content title="vLLM" level="h3" expanded=false id="vllm" %}}
{{< tabs >}}
{{% tab "Python" %}}

> The vLLM integration automatically traces request processing and token generation in the [vLLM][1] inference engine, capturing model name, input and output tokens, and latency metrics (time to first token, queue time, prefill time, decode time, inference time).

Reviewer: The parenthetical list is missing a serial comma before the final item. Also consider splitting the sentence — at 47 words it's on the long side.

```suggestion
The vLLM integration automatically traces request processing and token generation in the [vLLM][1] inference engine. It captures model name, input and output tokens, and latency metrics (time to first token, queue time, prefill time, decode time, and inference time).
```

> ### Traced methods

Reviewer: Same as above — the vLLM section is missing the `**Package name:**` and `**Integration name:**` metadata block.

```suggestion
**Package name:** `vllm`
**Integration name:** `vllm`

### Traced methods
```

Is this right?
